flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/catalog/CatalogViewITCase.scala

```diff
@@ -307,6 +307,91 @@ class CatalogViewITCase(isStreamingMode: Boolean) extends AbstractTestBase {
     // now we only have permanent view T3
     assertEquals(permanentViewData.sorted, TestCollectionTableFactory.RESULT.sorted)
   }
+
+  @Test
+  def testShowCreateView(): Unit = {
+    val sourceData = List(toRow(1, "1000", 2))
+    TestCollectionTableFactory.initData(sourceData)
+    val tableDDL: String =
+      s"""CREATE TABLE T1 (
+         |a int,
+         |b varchar,
+         |c int
+         |) with (
+         |'connector' = 'COLLECTION'
+         |)""".stripMargin
+    val tView1DDL: String = "CREATE TEMPORARY VIEW t_v1 AS SELECT a, b, c FROM T1"
+    val tView2DDL = "CREATE TEMPORARY VIEW t_v2(d, e, f) AS SELECT a, b, c FROM T1"
+    val view1DDL = "CREATE VIEW v1 AS SELECT a, b, c FROM T1"
+    val view2DDL = "CREATE VIEW v2(x, y, z) AS SELECT a, b, c FROM T1"
+    tableEnv.executeSql(tableDDL)
+    tableEnv.executeSql(tView1DDL)
+    tableEnv.executeSql(tView2DDL)
+    tableEnv.executeSql(view1DDL)
+    tableEnv.executeSql(view2DDL)
+    testShowCreateTemporaryView(tableEnv)
+    testShowCreateView(tableEnv)
```

Comment:
The split of methods here is really odd, because none of these methods are actually parameterized. I think this would be nicer to read if we organize the test like this:

```
// … setup code …

// Temporary views
// … code to set up first view …
// … assert first view …

// … code to set up second view …
// … assert second view …

// Catalog views
// … code to set up first view …
// … assert first view …

// … code to set up second view …
// … assert second view …
```

If anything, we could split it between temporary and catalog views (into two tests), that'd be a logical way to split this up a bit.

Reply:
Good idea. 
Done.